### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by L2oukmane**
### Re-create Awesome books app following:

- Divide the main JS file into modules.
- Respecting ES6 syntax.
- Use Luxon library.
- Keep all funcionalities of the app without errors.

### CODE REVIEW

File: Modules/navigate.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const navItems = Array.from(document.querySelectorAll('.navItems')[0].children);
+const header = document.querySelector('.books-heading');
+const newBook = document.querySelector('.add-books');
+const contact = document.querySelector('.contact-section');
+const booksWrapper = document.querySelector('.books-list');
+
+function navigate(key) {
```
Comment (ShoiraTa) [inline Modules/navigate.js]: - [ ] Please use ES6 arrow functions, eg.: 
`const navigate = (key) => { }`

File: Modules/BooksdataBaseClass.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable consistent-return */
```
Comment (ShoiraTa) [inline Modules/BooksdataBaseClass.js]: - [ ] Please avoid making custom changes to eslint

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
-# Awesome-Book
\ No newline at end of file
+# AWESOME-BOOKS | MODULE-II
+
+> Awesome Book is simple application being built in our second module of our curriculum at microverse. its a book application were you are able to add and remove favourite book to a book list.
```
Comment (ShoiraTa) [inline README.md:3]: - [ ] Professional README, well done 👏🏻

**Review by ShoiraTa — COMMENTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @L2oukmane ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## Optional suggestions

- [ ] OPTIONAL: I suggest making more frequent commits with shorter commit text in Imperative mood. 
For this milestone you have only one commit with a lot of text half of which is not visible. This practice will make it difficult to debug projects. 

- [ ] OPTIONAL: I suggest giving pull request shorter title, currently half of the PR title is not visible. You can always use project milestone title, in this case it is `Awesome books: with ES6` 😉 

Screenshot 2022-03-21 at 21 11 50

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @L2oukmane, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**